core/src/integration-test/java/com/scalar/db/storage/StorageMultiplePartitionKeyIntegrationTestBase.java

```diff
@@ -55,13 +61,17 @@
 
   private static long seed;
 
+  private static ExecutorService executorService;
```

Comment:
I also made this multiple partition key scan integration test run concurrently with multi threads.